pkg/resources/resources_manager.go

```diff
@@ -30,6 +30,9 @@ const (
 var (
 	activeSockDir     = "/var/lib/kubelet/plugins_registry"
 	deprecatedSockDir = "/var/lib/kubelet/device-plugins"
+
+	// Required RDMA devices
+	rdmaDevices = []string{"issm", "rdma_cm", "umad", "uverbs"}
```

Comment:
why not have this in `rdma_device_spec.go` and just use it for `VerifyRdmaSpec()` ?

this will make the change of this PR only in `rdma_device_spec.go` and `pci_net_device.go`

resource_manager should not have this info IMO.